lib/matplotlib/animation.py

```diff
@@ -1013,6 +1013,22 @@ class to use, such as 'ffmpeg'. If ``None``, defaults to
            on to the `savefig` command which is called repeatedly to
            save the individual frames.
 
+        progress_callback : function, optional
+            A callback function that will be called for every frame to notify
+            the saving progress. It must have the signature ::
+
+                def func(current_frame: int, total_frames: int) -> Any
+
+            where *current_frame* is the current frame number and
+            *total_frames* is the total number of frames to be saved.
+            *total_frames* is set to 0, if the total number of frames can
```

Comment:
In analogy to `save_count`, it would be better if this was `None` if the number of frames cannot be determined.

Reply:
I'm not familiar with these rst files, but to me the current rst files in that directory look quite inconsistent ... I'll add one, but that might not match the required style